docs/UserGuide.md

```diff
@@ -143,6 +143,45 @@ To aid the user, this command starts the interactive tutorial for either the [Ci
 
 * `TYPE` can be either `circuit` or `boolean`.
 
+Once the user has entered the interactive tutorial, they are to follow the instructions provided to them as follows:
+
+
+	You have entered
+	  _______    _             _       _   __  __           _      _
+	 |__   __|  | |           (_)     | | |  \/  |         | |    | |
+	    | |_   _| |_ ___  _ __ _  __ _| | | \  / | ___   __| | ___| |
+	    | | | | | __/ _ \| '__| |/ _` | | | |\/| |/ _ \ / _` |/ _ \ |
+	    | | |_| | || (_) | |  | | (_| | | | |  | | (_) | (_| |  __/_|
+	    |_|\__,_|\__\___/|_|  |_|\__,_|_| |_|  |_|\___/ \__,_|\___(_)
+
+	Type 'exit' if you want to leave this mode and go back to the application.
+
+	Let's get started! First we will choose a template to work with. In the app you can choose from rc, rl, lc, and r templates.
+	However, for this tutorial, we will use a Resistor-Capacitor template.
+	To select the Resistor-Capacitor circuit template, enter 'template rc'
+When the user has entered the tutorial, they will be required to type in the commands in the terminal
```

Comment:
This is buggy, consider adding new lines after code, as well as using ``` instead of tabs.
![image](https://user-images.githubusercontent.com/53231968/98352148-99887580-2058-11eb-8d96-3744335af7dd.png)

Reply:
Changed.

Reply:
Ah another thing, this looks like it is specific to `tutorial circuit` which is not mentioned anywhere.

Reply:
Okay, I have included it